state/migration_export.go

```diff
@@ -1177,15 +1177,10 @@ func (e *exporter) subnets() error {
 			ProviderNetworkId: string(subnet.ProviderNetworkId()),
 			VLANTag:           subnet.VLANTag(),
 			SpaceName:         subnet.SpaceName(),
+			AvailabilityZones: subnet.AvailabilityZones(),
 			FanLocalUnderlay:  subnet.FanLocalUnderlay(),
 			FanOverlay:        subnet.FanOverlay(),
 		}
-		// TODO(babbageclunk): at the moment state.Subnet only stores
-		// one AZ.
-		az := subnet.AvailabilityZone()
-		if az != "" {
-			args.AvailabilityZones = []string{az}
-		}
```

Comment:
So for migrations, have they always supported zones as a list and we just only ever transferred a single one? (so no compatibility issues with changing the stream?)